<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatClientClass</class>
 <widget class="QMainWindow" name="ChatClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InetChat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="StartW">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="start_app_button">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Start application</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LogInW">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <spacer name="verticalSpacer_top">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_right">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_left">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QWidget" name="nicknameW" native="true">
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="login_nickname_lable">
             <property name="text">
              <string>Nickname</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="login_nickname_edit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QWidget" name="buttonsW" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_buttonW">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="login_sign_in_button">
             <property name="text">
              <string>Sign in</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="login_log_in_button">
             <property name="text">
              <string>Log in</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QWidget" name="passwordW" native="true">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="password_lable">
             <property name="text">
              <string>Password </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="login_password_edit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_bottom">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ProfileW">
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="4,0,1,5,0">
           <item>
            <widget class="QLabel" name="profile_image_lable">
             <property name="text">
              <string>image_lable</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="profile_change_avatar_button">
             <property name="text">
              <string>Change Avatar</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="profile_raiting_text">
             <property name="text">
              <string>rating_lable</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="profile_start_chating_button">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Start chating</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,3,3,0,7,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <spacer name="verticalSpacer_data_top">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="profile_nickname_lable">
                <property name="text">
                 <string>Nickname</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="profile_nickname_edit"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QFormLayout" name="formLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="profile_password_lable">
                <property name="text">
                 <string>Password </string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="profile_password_edit"/>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignRight">
            <widget class="QPushButton" name="profile_change_password_button">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>ChangePassword</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_data_bottom">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item alignment="Qt::AlignRight">
            <widget class="QPushButton" name="profile_save_button">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ChatListW">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="chatList_add_chat_button">
             <property name="text">
              <string>Add chat</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="chatList_profile_button">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Profile</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="ChatWView" name="chatList_listView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ChatW">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="chat_name_lable">
             <property name="text">
              <string>Chat_name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="chat_exit_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 140, 129)</string>
             </property>
             <property name="text">
              <string>Exit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="MessageWView" name="chat_listView"/>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextEdit" name="text_edit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="send_button">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="add_attach_button">
                <property name="text">
                 <string>Add attach</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AddChatRoomEdit">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QWidget" name="widget_10" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Name         </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="add_room_name_edit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_11" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="add_room_descr_edit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_13" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Topic name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="add_room_combo_box"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="add_room_private_check_box">
          <property name="text">
           <string>Private</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_12" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>61</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Password   </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="add_room_password_edit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_9" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="add_room_cancel_button">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="add_room_create_button">
             <property name="text">
              <string>Create</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageWView</class>
   <extends>QListView</extends>
   <header>messagewview.h</header>
  </customwidget>
  <customwidget>
   <class>ChatWView</class>
   <extends>QListView</extends>
   <header>chatwview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
